lib/datasets.go

```diff
@@ -597,7 +598,15 @@ func (r *DatasetRequests) Remove(p *RemoveParams, res *RemoveResponse) error {
 				if wdErr == fsi.ErrWorkingDirectoryDirty {
 					return fmt.Errorf("cannot remove from dataset while working directory is dirty")
 				}
-				return wdErr
+				if strings.Contains(wdErr.Error(), "not a linked directory") {
+					// If the working directory has been removed (or renamed), could not get the
+					// status. However, don't let this stop the remove operation, since the files
+					// are already gone, and therefore won't be removed.
+					log.Debug("could not get stauts for %s, maybe removed or renamed", ref.FSIPath)
```

Comment:
status -> status

`log.Debug` should be `log.Debugf`. go vet won't catch this because the log package isn't directly invoking `fmt`